hexrd/ui/resources/ui/mask_manager_dialog.ui

```diff
@@ -19,6 +19,15 @@
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
+     <property name="editTriggers">
+      <set>QAbstractItemView::DoubleClicked</set>
+     </property>
+     <property name="alternatingRowColors">
+      <bool>true</bool>
+     </property>
+     <property name="sortingEnabled">
+      <bool>true</bool>
+     </property>
```

Comment:
Can you also set the `selectionBehavior` to be `SelectRows`? I think that is more intuitive for this table especially because the context menu is essentially acting on a whole row, rather than a single item (which includes the particular column that was selected).

Reply:
Yes, the behavior has been updated.